packages/studio/src/utils/codemirror.utils.ts

```diff
@@ -31,13 +29,15 @@ export const myTheme = EditorView.theme({
     }
 }, {dark: true})
 
-export const language = new Compartment 
+export const language = new Compartment
 
 export const myHighlightStyle = HighlightStyle.define([
-  {tag: customTags.chord, color: "#0481aa"},
+  {tag: tags.keyword, color: "#0481aa"},
   {tag: tags.bracket, color: "#fc6"},
-  {tag: tags.meta, color: "#fa7c6c"},
-  {tag: tags.comment, color: "#5d6063"}
+  {tag: tags.attributeName, color: "#8a8c8d"},
+  {tag: tags.attributeValue, color: "#fa7c6c"},
+  {tag: tags.lineComment, color: "#5d6063"},
+  {tag: tags.content, color: '#fff'}
```

Comment:
Small thing but I think white's a harsh color over dark background. I think you should set this to #c1c1c1. Just a bit easier to read in the meantime.